pkg/live/inventoryrg.go

```diff
@@ -34,7 +34,7 @@ import (
 // resource used to store inventory.
 var ResourceGroupGVK = schema.GroupVersionKind{
 	Group:   "kpt.dev",
-	Version: "v1alpha1",
+	Version: "v1alpha2",
```

Comment:
Why are we updating the version of the RG CRD? I don't think there is a need to keep this version aligned with the version in the Kptfile.